views/index.hbs

```diff
@@ -8,25 +8,25 @@
 	</button>
 </form>
 {{#if entries}}
-	<ul class="entry-list">
-		{{#each entries}}
-			<li class="entry">
-				<h2 class="entry__question">
-					{{this.question}}
-				</h2>
-				<div class="entry__answer">
-					{{{this.answer}}}
-				</div>
-				<div class="entry__help">
-					{{this.helpful}}/?? people found this helpful. Was this entry helpful?
-					<button class="button button--small">Yes</button>
-					<button class="button button--small">No</button>
-				</div>
-			</li>
-		{{/each}}
-	</ul>
+<ul class="entry-list">
+	{{#each entries}}
+	<li class="entry">
+		<h2 class="entry__question">
+			{{this.question}}
+		</h2>
+		<div class="entry__answer">
+			{{{this.answer}}}
+		</div>
+		<div class="entry__help">
+			<span class="value-helpful" data-question-id={{this._id}}>{{this.helpful}} people found this helpful. Was this entry helpful?</span>
```

Comment:
Missing quotes `"`  around `{{this._id}}`.

Also, you're missing the total count: 11/25 people found this helpful. You may need @khaledkzy to modify the data delivered to the template for you.

Reply:
Khalid accept only the ID and the boolean of the Helpful buttons, and he changed depends on what he gets from me
if it's false he add the total number and if it's true he adds 1 to the helpful

Reply:
Yes, but in this initial template, the string displays without a total count. You only say "11 people found this helpful" but you need to display "11/25 people found this helpful". Yeah?